src/pages/Admin/AdminUpdateClass/ClassComponent.tsx

```diff
@@ -1,66 +1,83 @@
 import React from 'react';
 
+import dateFormat from 'dateformat';
 import {
   StyledClassContainer,
   StyledClassHeaderWrapper,
   ClassDate,
   ClassDescription,
   ClassTitle,
+  AddClassContentIcon,
   ClassTitleWrapper,
-  EditClassContentIcon,
   EditClassIcon,
   StyledClassItem,
   StyledClassItemsContainer,
   TrashIcon,
 } from './ClassComponent.styled';
-import { ClassComponentData } from '../../../api/ClassComponentData';
 import AccordionWrapper from '../../../components/Accordion/Accordion';
 
 interface ClassComponentDataProps {
-  id: string;
+  _id: string;
   date: string;
-  title: string;
-  classDescription: string;
-  content?: [];
+  name: string;
+  subject: string;
+  classworks?: [];
+}
+
+interface classesProps {
+  classes: ClassComponentDataProps[] | undefined;
 }
 
 const getHeaderComponent = (item: ClassComponentDataProps) => {
   return (
     <StyledClassHeaderWrapper>
       <article>
-        <ClassDate>{item.date}</ClassDate>
+        <ClassDate>{dateFormat(`${item.date}`, 'mmmm dS, yyyy')}</ClassDate>
```

Comment:
hmm I think another library might not be needed? I got the formatting to work using 
```
new Intl.DateTimeFormat(lang, options).format(
    new Date(year, month, day)
  )
```
see `CohortCard` component for more details